pkg/resources/fluentbit/configsecret.go

```diff
@@ -126,7 +126,7 @@ func (r *Reconciler) configSecret() (runtime.Object, reconciler.DesiredState, er
 
 	var fluentbitTargetHost string
 	if r.Logging.Spec.FluentdSpec != nil && r.Logging.Spec.FluentbitSpec.TargetHost == "" {
-		fluentbitTargetHost = fmt.Sprintf("%s.%s.svc.cluster.local", r.Logging.QualifiedName(fluentd.ServiceName), r.Logging.Spec.ControlNamespace)
+		fluentbitTargetHost = fmt.Sprintf("%s.%s.svc.%s", r.Logging.QualifiedName(fluentd.ServiceName), r.Logging.Spec.ControlNamespace, r.Logging.Spec.ClusterDomain)
```

Comment:
I think this format string will have the same problem with the trailing dot if `ClusterDomain` is set to empty as I noted earlier with the chart template.

Reply:
I imagined the field could not be set to empty, either it takes the specified value, or it uses the defaulted value from https://github.com/banzaicloud/logging-operator/pull/1031/files#diff-911a398d9218f00dc5749daedc4d64b64f16f2f8b010c898dc28393b3454fd0dR143

WDYT?

Reply:
By leaving off the tail of the FQDN the user can have more flexibility in how domain queries are resolved, utilizing the `search` option of `resolv.conf` (e.g. through `spec.dnsConfig.searches` field of the Pod resource). I think, it would be a shame not to support this given how little effort it would take on our side. Also, I see no downside in supporting this, but I might be wrong. Do you have any arguments against?

Reply:
Looks good to me. Will work on this.